pyretis/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pyretis" %}
+{% set version = "0.9.2.dev2" %}
+{% set sha256 = "06c73d2778de7c65ae30464e2456637dfd326c9d94fd0bbfd557a789bff87816" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py<3]
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  ## preserve_egg_dir: True
+
+requirements:
+  build:
+  - python
+  - setuptools
+  - colorama
+  - tqdm
+  - future
+  - numpy
+  - scipy
+  - matplotlib
+  - jinja2
```

Comment:
It looks like none of these are actually needed for building the package, so you should be able to delete everything but `python` and `setuptools` from here.